app/policies/application_policy.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+    raise Pundit::NotAuthorizedError, 'must be logged in to Panoptes' unless user
```

Comment:
Won't this cause _all_ requests that can't authorize a user to raise this Pundit error? Every controller inherits from ApplicationController, and therefore will apply the ApplicationPolicy. Since this is in `#initialize` and not a specific inherited method, this will bounce any non-logged in users away from not only the `/classifications/users/*` routes, but also the `/classifications/*` ones that are meant to be available to non-logged-in users also.

Reply:
Right..I had `skip_authorization`s on a lot of the endpoints that are available to non-logged-in users and thought it would be safer to add this in case an unauthenticated user gets through; but this most likely is not necessary in stats case. yeah.. i'll remove this.